src/inmanta_ui/ui.py

```diff
@@ -63,16 +63,18 @@ def add_web_console_handler(self, server: Server) -> None:
         if path is None:
             return
 
+        config_js_content = ""
         if opt.server_enable_auth.get():
-            auth = f"""
+            config_js_content = f"""
         window.auth = {{
             'realm': '{opt.dash_realm.get()}',
             'url': '{opt.dash_auth_url.get()}',
             'clientId': '{opt.dash_client_id.get()}'
-        }};"""  # Use the same client-id as the dashboard
-        else:
-            auth = ""
-        server.add_static_content("/console/config.js", content=auth)
+        }};\n"""  # Use the same client-id as the dashboard
+        if web_console_json_parser.get() == "BigInt":
+            config_js_content += f"window.jsonParserId = '{web_console_json_parser.get()}';\n"
```

Comment:
We need to add validation to the config option. Because with this you can actually include arbitrary code

Reply:
Well it's checked whether the value of the option is the `BigInt` literal. Or do you mean the `Option` definition in `config.py` should be stricter than `str`?

Reply:
no, you are right. I misread the code.